### PR TITLE
Update to Jetty 9.4.24.v20191120

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -1,3 +1,7 @@
+0.188
+
+- Remove status message support from HTTP client.
```
Comment (martint) [inline CHANGES]: What's the motivation?

File: http-client/src/test/java/io/airlift/http/client/AbstractHttpClientTest.java [hunk 2]
```diff
@@ -717,7 +717,7 @@ public void testCompressionIsDisabled()
         assertEquals(body, "");
         assertFalse(servlet.getRequestHeaders().containsKey(HeaderName.of(ACCEPT_ENCODING)));
 
-        String json = "{\"foo\":\"bar\",\"hello\":\"world\"}";
+        String json = "{\"fuite\":\"apple\",\"hello\":\"world\"}";
```
Comment (martint) [inline http-client/src/test/java/io/airlift/http/client/AbstractHttpClientTest.java]: Unrelated change?

File: CHANGES (hunk 1 above)
Comment (dain) [reply CHANGES]: It isn't really supported anymore in the Jetty clients, and for HTTP/2 you just get `null`.  Also we never use it.

Comment (martint) [reply CHANGES]: Ok, maybe we should add that to the commit message

File: http-client/src/test/java/io/airlift/http/client/AbstractHttpClientTest.java (hunk 2 above)
Comment (electrum) [reply http-client/src/test/java/io/airlift/http/client/AbstractHttpClientTest.java]: No, the minimum size for gzip increased.
